src/v2/utils/parser/RozParser.ts

```diff
@@ -180,6 +183,24 @@ export class RozParser implements Parser {
     }
   }
 
+  async getTeacherFullIniatials (lastName, firstName, middleName) {
+    if (firstName.length <= 1 || middleName.length <= 1) {
+      const teachers = await this.prisma.teacher.findMany({ 
```

Comment:
Create after that issue where this code must be written through teacher repository

Reply:
Opened https://github.com/fictadvisor/fictadvisor-api/issues/302